### ISSUE TITLE
A few issues with Coc

### ISSUE THREAD

**Issue by Avi-D-coder** [vhakulinen/gnvim#105]
#95 will likely fix layout, but the little black lines between each line of text appear to be separate problem.
![Screenshot_20190905_175651](https://user-images.githubusercontent.com/29133776/64386406-05132500-d007-11e9-99dc-78721c28f0c3.png)

**Comment by smolck** [comment]
Hello! Thanks for reporting this issue! I believe that this is a duplicate of #85 (I’ve been seeing it with the new pango 1.44), which does have a workaround: https://github.com/vhakulinen/gnvim/issues/85#issuecomment-527200720

Would you mind either trying that diff or downgrading pango to v1.43 to see if that fixes this issue? Note that there are black lines (more pronounced) in the second image of the opening comment of #85.

**Comment by Avi-D-coder** [comment]
Yep your right the lines do look like a duplicate. Will test to make sure latter. Thanks

**Comment by vhakulinen** [comment]
Could you try with `--disable-ext-popupmenu`?

**Comment by Avi-D-coder** [comment]
`--disable-ext-popupmenu` does appear to fix the layout.

**Comment by vhakulinen** [comment]
Note to self: there seems to be `nvim_ui_pum_set_bounds` function which might help with this.
